### PR TITLE
Added docker compose

### PR DESCRIPTION
**Pull request by Vovchyk**
Docker-compose for services involved in flyover

### CODE REVIEW

File: docker-compose/.env [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
```
Comment (josedahlquist) [inline docker-compose/.env:1]: Should we push a .env.test file instead?

File: docker-compose/lbc-deployer/deploy.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+if [[ -d $CONTRACTS_DIR && "$(ls -A $CONTRACTS_DIR)" ]]; then
+    echo "Cleaning up $CONTRACTS_DIR directory..."
+    rm $CONTRACTS_DIR/*.*
+fi
+
+echo "Deploying contracts to RskJ..."
+
+cd /code/lbc
+
+npx truffle deploy --network rskRegtest
```
Comment (josedahlquist) [inline docker-compose/lbc-deployer/deploy.sh:21]: So this will be just for regtest?

File: docker-compose/rskj/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+FROM openjdk:8-jdk as runner
+
+RUN useradd -m rsk
+
+WORKDIR /home/rsk
+
+USER rsk
+COPY --from=builder --chown=rsk:rsk /code/rskj/rskj-core/build/libs/rskj-core-* ./
+
+ENTRYPOINT ["java", "-Drpc.providers.web.http.bind_address=0.0.0.0", "-Drpc.providers.web.http.hosts.0=localhost", "-Drpc.providers.web.http.hosts.1=rskj", "-cp", "rskj-core-3.1.0-IRIS-all.jar", "co.rsk.Start"]
```
Comment (josedahlquist) [inline docker-compose/rskj/Dockerfile:30]: this is starting a mainnet node, right?

**Review by josedahlquist — COMMENTED** [review]
I'm a bit confused here... remind me, what's the purpose of the containerization? because I see some things that makes me thing it's is for testing purpose and other that this could be intenteded for mainnet...?

**Comment by Vovchyk** [comment]
> I'm a bit confused here... remind me, what's the purpose of the containerization? because I see some things that makes me thing it's is for testing purpose and other that this could be intenteded for mainnet...?

I believe the initial idea was to have a capability to easily spin up local env. Im now thinking that it'd be nice to make use of it for running a few integration tests (e.g. in the form of gihub actions), which would include a couple of use cases of quote generation and acceptance. What do you think?

File: docker-compose/.env (hunk 1 above)
Comment (Vovchyk) [reply docker-compose/.env:1]: do you see some other way of using it? For now, we don't have to provide `--env-file` arg for docker-compose command, as that's a default name

File: docker-compose/lbc-deployer/deploy.sh (hunk 2 above)
Comment (Vovchyk) [reply docker-compose/lbc-deployer/deploy.sh:21]: yeah, the idea was to spin up a local environment with regtest  instances of rsk and bitcoin

File: docker-compose/rskj/Dockerfile (hunk 3 above)
Comment (Vovchyk) [reply docker-compose/rskj/Dockerfile:30]: well, it's just an entry point. The network flag is being provided in the docker-compose.yml file [here](https://github.com/rsksmart/liquidity-provider-server/pull/36/files#diff-21796814c2204c04ba29b3834e57d5399c2ba04589de42b2be845cf6a0309b11R24)

**Comment by josedahlquist** [comment]
I think having integration tests for this would be really great. For the time we can leave it as is and move on. Thanks Vlad!

**Review by josedahlquist — APPROVED** [review]

**Review by josedahlquist — APPROVED** [review]

**Pull request merged**